bork/cli.py

```diff
@@ -40,6 +40,14 @@ def clean():
     api.clean()
 
 
+@cli.command()
+@click.option('-o', '--output', type=click.File('w'), default='-',
+              help='File in which to save the list of dependencies.')
```

Comment:
So, `print(string, file='-')` doesn't work &mdash; is accepting `-` a magical click thing? If so, that's pretty neat.

Reply:
nvm, I [r'd tfm](https://click.palletsprojects.com/en/7.x/arguments/#file-arguments) and yup it's a magical click thing.

Reply:
Yeah, it's a [click thing](https://click.palletsprojects.com/en/7.x/arguments/#file-arguments)